torch/onnx/verification.py

```diff
@@ -132,6 +133,10 @@ def _compare_ort_pytorch_outputs(
         pt_outs: outputs from PyTorch.
         rtol (float, optional): relative tolerance in comparison between ONNX and PyTorch outputs.
         atol (float, optional): absolute tolerance in comparison between ONNX and PyTorch outputs.
+        ignoreNone (bool, optional): Default True. If True, ignore None type in
```

Comment:
When writing docstring for booleans, use: "Whether to..."

> Whether to ignore None values in the torch output, ...

Mention the default in the end. Match the behavior in the signature since currently it is required.

Reply:
Done